pkg/cloud/azure/storageconnection.go

```diff
@@ -46,6 +47,29 @@ func (sc *StorageConnection) getBlobURLTemplate() string {
 	return "https://%s.blob.core.windows.net/%s"
 }
 
+// DownloadBlob downloads the Azure Billing CSV into a byte slice
```

Comment:
I've brought this method back exactly as-is based on Thomas' [commit](https://github.com/opencost/opencost/pull/2456/commits/7b984319fc5ca103a13c60111e9ee4b2b113f04b) part of the original [PR](https://github.com/opencost/opencost/pull/2456)